omnisci_olio/dashboard/dashboard_utils.py

```diff
@@ -98,6 +95,7 @@ def read_dashboard(self, filename) -> Dict[str, Any]:
                     td['state'] = json.loads(all)
                     return td
         except Exception as e:
+            L.error(f'Exception occurs: {e}')
```

Comment:
logging here is not necessary since it's re-raised. Logging would add duplicate error output

Reply:
Yep, I agree.